be/src/util/value_generator.h

```diff
@@ -0,0 +1,22 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#pragma once
+#include "runtime/date_value.h"
+#include "runtime/timestamp_value.h"
+
+namespace starrocks::vectorized {
+template <class T>
+struct DefaultValueGenerator {
+    static T next_value() { return T(); }
```

Comment:
The default value is always one value. why need next value or generator?

Reply:
I was thinking that later we might write some data generators. It is possible to use them in unit tests or goolgle-benchmark. For example, data generators for certain ranges, data generators for certain distributions.